staging/src/k8s.io/csi-translation-lib/translate.go

```diff
@@ -76,18 +76,27 @@ func TranslateCSIPVToInTree(pv *v1.PersistentVolume) (*v1.PersistentVolume, erro
 }
 
 // IsMigratableByName tests whether there is Migration logic for the in-tree plugin
-// for the given `pluginName`
-func IsMigratableByName(pluginName string) bool {
+// whose name matches the given inTreePluginName
+func IsMigratableByName(inTreePluginName string) bool {
 	for _, curPlugin := range inTreePlugins {
-		if curPlugin.GetInTreePluginName() == pluginName {
+		if curPlugin.GetInTreePluginName() == inTreePluginName {
 			return true
 		}
 	}
 	return false
 }
 
-// GetCSINameFromIntreeName maps the name of a CSI driver to its in-tree version
-func GetCSINameFromIntreeName(pluginName string) (string, error) {
+// SupersedesInTreePlugin tests whether there exists an in-tree plugin with logic
+// to migrate to the CSI plugin with given csiPluginName
+func SupersedesInTreePlugin(csiPluginName string) bool {
```

Comment:
We need to come up with better names (all of us in general, not just this). 

This is the CSI direction of `IsMigratableByName` and I think it should be evident through the naming.
Maybe they should be `IsIntreePluginMigratableByName` and `IsCSIDriverMigratableByName` instead?
WDYT?

Naming things is hard.

Reply:
API names are indeed hard! The one thing that bothers me about `IsCSIDriverMigratableByName` is: the CSI Driver is not migrat"able" to something - it's what the in-tree plugin has been migrat"ed" to. 

I was leaning towards `IsMigratedCSIDriverByName` and `IsMigratableIntreePluginByName` but I guess that does not align with the desired convention. If not, I will go ahead with `IsIntreePluginMigratableByName` and `IsCSIDriverMigratableByName`

Reply:
I think your suggestion (IsMigratedCSIDriverByName and IsMigratableIntreePluginByName) makes sense. Thanks :+1: